tests/models/test_models_vae.py

```diff
@@ -383,3 +433,149 @@ def test_stable_diffusion_encode_sample(self, seed, expected_slice):
 
         tolerance = 3e-3 if torch_device != "mps" else 1e-2
         assert torch_all_close(output_slice, expected_output_slice, atol=tolerance)
+
+
+@slow
```

Comment:
Which GPU was used to gather the expected values? What was the testing environment? We run the SLOW tests on a Tesla V100 and to get a sense of the deb environment, I'd suggest using Docker (ID: `diffusers/diffusers-pytorch-cuda`) and the dependencies we use here: https://github.com/huggingface/diffusers/blob/main/.github/workflows/push_tests.yml

Reply:
yup, it was tested with `diffusers-pytorch-cuda`

Reply:
I'm using rtx3090

Reply:
Hmm. Let me try to get those numbers for V100.

Reply:
I am seeing the following during the SLOW tests:

```
Some weights of the model checkpoint were not used when initializing AsymmetricAutoencoderKL: 
 ['decoder.condition_encoder.up_layers.0.deconv.weight, decoder.condition_encoder.up_layers.0.deconv.bias, decoder.condition_encoder.up_layers.1.deconv.weight, decoder.condition_encoder.up_layers.1.deconv.bias, decoder.condition_encoder.up_layers.2.deconv.weight, decoder.condition_encoder.up_layers.2.deconv.bias, decoder.condition_encoder.up_layers.3.deconv.weight, decoder.condition_encoder.up_layers.3.deconv.bias']
```

Is that expected? 

Additionally, the following SLOW tests are failing on V100 (which is used to run our CI after a PR is merged into `main`):

```
FAILED tests/models/test_models_vae.py::AsymmetricAutoencoderKLIntegrationTests::test_stable_diffusion_0 - AssertionError: Max diff is absolute 0.003887087106704712. Diff tensor is tensor([1.1746e-04, 3.8871e-03, 6.8235e-04, 3.3728e-04, 3.3337e-...
FAILED tests/models/test_models_vae.py::AsymmetricAutoencoderKLIntegrationTests::test_stable_diffusion_decode_1 - AssertionError: Max diff is absolute 0.0010186508297920227. Diff tensor is tensor([0.0001, 0.0002, 0.0010, 0.0002, 0.0004, 0.0003, 0.0006,...
```

Here are the outputs I am getting from the tests (I guess we can update with these values?):

```
From SD:
output_slice: tensor([-0.0343,  0.2873,  0.1680, -0.0140, -0.3459,  0.3522, -0.1336,  0.1075])
expected output slice: tensor([-0.0344,  0.2912,  0.1687, -0.0137, -0.3462,  0.3552, -0.1337,  0.1078])

Decode: 
output_slice: tensor([-0.1821, -0.4347, -0.0465, -0.2925, -0.8031, -0.5086, -0.4789, -0.3100])
expected_output_slice: tensor([-0.1820, -0.4345, -0.0455, -0.2923, -0.8035, -0.5089, -0.4795, -0.3106])
```

Let me know if anything is unclear.

Reply:
@sayakpaul
updated values and models

Reply:
SLOW tests are fully passing now. Thanks!